app/src/main/res/layout/item_vertical_image.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
File is now called item_row_with_title (mege conflict). can be deleted as a whole